MigrateMail.pm

```diff
@@ -0,0 +1,46 @@
+package MigrateMail;
```

Comment:
Please add a copyright header:

# copyright 2017 cpanel, Inc
# Licensed under the terms of the Apache 2 license